interface/billing/billing_report.php

```diff
@@ -77,43 +75,45 @@
             header("Content-Disposition: attachment; filename=openemr_ofx.ofx");
             header("Content-Type: text/xml");
             echo $ofx->get_OFX();
-            exit;
+            exit();
         }
     }
 }
 
-//global variables:
-$from_date     = isset($_POST['from_date'])  ? $_POST['from_date']  : date('Y-m-d');
-$to_date       = isset($_POST['to_date'  ])  ? $_POST['to_date'  ]  : '';
-$code_type     = isset($_POST['code_type'])  ? $_POST['code_type']  : 'all';
-$unbilled      = isset($_POST['unbilled' ])  ? $_POST['unbilled' ]  : 'on';
+// global variables:
+$from_date = isset($_POST['from_date']) ? $_POST['from_date'] : date('Y-m-d');
+$to_date = isset($_POST['to_date']) ? $_POST['to_date'] : '';
+$code_type = isset($_POST['code_type']) ? $_POST['code_type'] : 'all';
+$unbilled = isset($_POST['unbilled']) ? $_POST['unbilled'] : 'on';
 $my_authorized = isset($_POST["authorized"]) ? $_POST["authorized"] : '';
 
-
 // This tells us if only encounters that appear to be missing a "25" modifier
 // are to be reported.
-$missing_mods_only = (isset($_POST['missing_mods_only']) && !empty($_POST['missing_mods_only']));
+$missing_mods_only = (isset($_POST['missing_mods_only']) && ! empty($_POST['missing_mods_only']));
 
 $left_margin = isset($_POST["left_margin"]) ? $_POST["left_margin"] : $GLOBALS['cms_left_margin_default'];
-$top_margin  = isset($_POST["top_margin"] ) ? $_POST["top_margin" ] : $GLOBALS['cms_top_margin_default'];
+$top_margin = isset($_POST["top_margin"]) ? $_POST["top_margin"] : $GLOBALS['cms_top_margin_default'];
 
-$ofrom_date  = $from_date;
-$oto_date    = $to_date;
-$ocode_type  = $code_type;
-$ounbilled   = $unbilled;
+$ofrom_date = $from_date;
+$oto_date = $to_date;
+$ocode_type = $code_type;
+$ounbilled = $unbilled;
 $oauthorized = $my_authorized;
 ?>
-
+<!DOCTYPE html >
 <html>
 <head>
 <?php if (function_exists('html_header_show')) html_header_show(); ?>
```

Comment:
Since we're here, let's go ahead and update this script to use the `Header` class